network/interface.go

```diff
@@ -38,7 +38,7 @@ type Transactions interface {
 	GetTransaction(transactionRef hash.SHA256Hash) (dag.Transaction, error)
 	// CreateTransaction creates a new transaction with the specified payload, and signs it using the specified key.
 	// If the key should be inside the transaction (instead of being referred to) `attachKey` should be true.
```

Comment:
maybe document the behaviour in relation to `additionalPrevs`

Reply:
added some info